src/Symfony/Component/Serializer/Normalizer/ObjectNormalizer.php

```diff
@@ -33,6 +34,10 @@ class ObjectNormalizer extends AbstractNormalizer
 
     public function __construct(ClassMetadataFactoryInterface $classMetadataFactory = null, NameConverterInterface $nameConverter = null, PropertyAccessorInterface $propertyAccessor = null)
     {
+        if (!class_exists('Symfony\Component\PropertyAccess\PropertyAccess')) {
+            throw new RuntimeException('Unable to use ObjectNormalizer as the Symfony PropertyAccess component is not installed.');
```

Comment:
What do you guys think about:

> The ObjectNormalizer class requires the "PropertyAccess" component. Install symfony/property-access to use it.

The "Install symfony/property-access" seems important now that even FW users are responsible in general for installing packages that they need.

Reply:
Good idea, however it will be many other texts to change (I've adapted an existing one, coming from FrameworkBundle IIRC).

Reply:
Done, I've just added extra quotes around `symfony/property-access`.